c/CMakeLists.txt

```diff
@@ -13,10 +13,7 @@ add_sources(nitf/shared/*.c)
 
 add_library(${NITF_C_LIB_NAME} SHARED ${SRC})
```

Comment:
should we do PRIVATE target_include_dir and target_link_libraries here too?

Reply:
I don't understand.  This is just creating the C library from source files.